src/main/java/com/internet/shop/web/filters/AuthorizationFilter.java

```diff
@@ -0,0 +1,86 @@
+package com.internet.shop.web.filters;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Role;
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebFilter("/*")
+public class AuthorizationFilter implements Filter {
+    private static final String USER_ID = "user_id";
```

Comment:
How about create USER_ID in one class and use it everywhere from that class?

Reply:
I don't know why in videos Bogdan didn't do it. Maybe there is a reason which I don't see. So I'm waiting for menthor's comment)

Reply:
@OleksiiTromsa @ilya449 both ways are ok, there is no strict requirement here)

Reply:
Yes, but I had a problem, when in several classes I had ```USER_ID = "userId"``` and in other ```USER_ID = "user_id"```. So I've decided to create one and user it everywhere